packaging/userspace/runpath_util.py

```diff
@@ -299,7 +302,7 @@ def _set_rpath_chrpath(
 				os.unlink(tmp_out)
 			op_state.chrpath.tool_tried = True
 
-	# FIXME: runpath setting with cmake is broken on centos
+	# FIXME: runpath setting with cmake fails silently on centos 7
```

Comment:
Is this something to worry about?

Reply:
Only if a an older version of lief is being used and `chrpath` is not present on the system.